x-pack/plugins/fleet/server/saved_objects/migrations/to_v8_7_0.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectMigrationFn } from '@kbn/core/server';
+
+import type { Installation } from '../../../common';
+import type { Output } from '../../../common/types';
+
+export const migrateInstallationToV870: SavedObjectMigrationFn<Installation, Installation> = (
+  installationDoc
+) => {
+  installationDoc.attributes.verification_status = 'unknown';
+
+  return installationDoc;
+};
+
+export const migrateOutputToV870: SavedObjectMigrationFn<Output, Output> = (
+  outputDoc,
+  migrationContext
+) => {
+  if (outputDoc.attributes.shipper) {
+    outputDoc.attributes.shipper.disk_queue_enabled = false;
+    outputDoc.attributes.shipper.disk_queue_path = '';
```

Comment:
Is this migration really necessary? given `shipper` is a newly introduced prop, we're basically doing a migration for a no-op here, unless I'm missing something?

Reply:
Yeah I was being overly protective here. I'll remove it for now and if necessary I'll add it back in the next PR.